web/src/main/webapp/js/instructions.js

```diff
@@ -46,7 +46,13 @@ function addInstruction(mapLayer, main, instr, instrIndex, lngLat, useMiles) {
                     setLatLng([lngLat[1], lngLat[0]]).
                     setContent(title).
                     openOn(mapLayer.getMap());
+
         });
+
+        // Debug Turn Instructions more easily
+        L.marker([lngLat[1], lngLat[0]], {
+            draggable: true
+        }).addTo(mapLayer.getRoutingLayer()).bindPopup(title);
```

Comment:
Can we always enable turn instruction debugging via an URL parameter `debug_instruction=true` or something? Also printing a marker for every instruction is really helpful (but not for the normal public audience of course ;))